packaging/linux/build_binaries.sh

```diff
@@ -114,12 +114,21 @@ build_one_architecture() {
 
   # Whitelist for NativeMessaging
   kbnm_bin="/usr/bin/kbnm"
+
+  # ... for Chrome:
   kbnm_file="$layout_dir/etc/opt/chrome/native-messaging-hosts/io.keybase.kbnm.json"
   mkdir -p "$(dirname "$kbnm_file")"
   cat "$here/host_json.template" \
     | sed "s|@@HOST_PATH@@|$kbnm_bin|g" \
     > "$kbnm_file"
 
+  # ... for Chromium:
+  kbnm_file="$layout_dir/etc/chromium/native-messaging-hosts/io.keybase.kbnm.json"
+  mkdir -p "$(dirname "$kbnm_file")"
+  cat "$here/host_json.template" \
+    | sed "s|@@HOST_PATH@@|$kbnm_bin|g" \
+    > "$kbnm_file"
```

Comment:
Could do that. I usually start generalizing when there are >2 uses of something, but it's a fair point.